ledger-bridge.js

```diff
@@ -22,36 +22,35 @@ export default class LedgerBridge {
     addEventListeners () {
         window.addEventListener('message', async e => {
             if (e && e.data && e.data.target === 'LEDGER-IFRAME') {
-                const { action, params } = e.data
-                const replyAction = `${action}-reply`
```

Comment:
Did you intend to remove this line? This variable is still referenced all over the place.